open-grants/accepted-open-grant-applications.md

```diff
@@ -5,15 +5,33 @@ This page provides an overview of accepted applications after July 1, 2021 and a
 ## Table of Contents <!-- omit in toc -->
 
 - [2021](#2021)
-  - [:surfing_woman: Wave 8](#surfing_woman---wave-8)
+
+  - [:surfing_woman: Wave 7](#surfing_woman---wave-8)
   
 ---
 
 # 2021
 
+## :surfing_woman: Wave 7
+
+| Team | Project | Link | Application Link | 
+| :--- | :------ | :--- | :--------: | 
+|    Filswan     | FS3    |   [Project](https://filswan.com/#/login)   |  [Application](https://github.com/filecoin-project/devgrants/blob/babea053951ae6cfa3334a8454f74b110ac1d115/open-grant-proposals/FS3_S3_compatible_storage_gateway.md)          |
+| Geo Web | Geo Web Network | [Project](https://www.geoweb.network/)  | [Application](https://github.com/filecoin-project/devgrants/blob/e4c96dcdd1bcd5ab82bd9e2d2fe5152b8f40a9e0/open-grant-proposals/open-proposal-geo-web.md)|
+| Xops  | OpenRPC | [Project](https://github.com/open-rpc/generator) | [Application](https://github.com/filecoin-project/devgrants/blob/b55e88ea50ecee739744c3571dcc7cb4724d2be8/open-grant-proposals/open-proposal-open-rpc-part-2.md)|
+| Go-graphsplit | Go-graphsplit | [Project](https://github.com/filedrive-team/go-graphsplit)| [Application](https://github.com/filecoin-project/devgrants/blob/8451afe585320d8401b0c404304d7f7afd859864/open-grant-proposals/open-proposal-Go-graphsplit.md)|
+| nanditmehra | Lighthouse pt.2 | [Project](http://ec2-13-126-82-18.ap-south-1.compute.amazonaws.com/) | [Application](https://github.com/nandit123/devgrants/blob/master/open-grant-proposals/lighthouse.md) |
+| kernelogic | Music NFT Portal | Project | [Application](https://github.com/filecoin-project/devgrants/blob/4267d83cf06219aec464ea143090480a089733da/open-grant-proposals/open-proposal-music-nft-portal.md) |
+| baoruian | IPFS SRE Community | [Project](https://www.kdocs.cn/view/l/srXkNrCpZAn2) | [Application](https://github.com/taoshengshi/devgrants/blob/master/open-grant-proposals/ipfs-sre-community.md) |
+|  |  |  |  |
+
+
+
+
 ## :surfing_woman: Wave 8
 
 | Team | Project | Link | Application Link | 
 | :--- | :------ | :--- | :--------: | 
-|      |         |      |            |
+| Wrapped  |  Bidirectional FIL <=> WFIL Bridge to Ethereum   |  [Project](https://github.com/tokensoft/tokensoft_token) | [Application](https://github.com/filecoin-project/devgrants/pull/264/commits/cf708760798428775091d092fbde21437db48df8?short_path=0c04afe#diff-0c04afe96ddd40ad02ecbb4037387a5842f2afdc0a3aa40e773fc5e6d7595f9b) |
+| wusirdcenter |  Improve the stability of miner | [Project](https://github.com/wusirdcenter)| [Application](https://github.com/filecoin-project/devgrants/pull/265/commits/93cac70de6e86021c06d1dadbcf03b7cab52b6be?short_path=e07ae8c#diff-e07ae8c83fac6e5d5b0acc35f8d9e61798b8b0c7c70ca0f97372b1cb1f3dc341)  |
```

Comment:
The project was renamed "Decoupling WinningPoSt and WindowPoSt from lotus-miner"